tree_goals:
  - {x: 1.0, y: 0.0}
  - {x: 2.0, y: 0.0}
  - {x: 3.0, y: 0.0}

arm_ee_goal:
  - [0.3, -0.4225, 0.4732, 0.9942, 0.0157, -0.106, 0.0180,  -0.9996,   0.0210,  -0.1056,  -0.0228,  -0.9941] 
  - [0.5271, 0.1225, 0.2732, 0.9942, 0.0157, -0.106, 0.0180,  -0.9996,   0.0210,  -0.1056,  -0.0228,  -0.9941] 
  - [0.3871, -0.4225, 0.5732, 0.9942, 0.0157, -0.106, 0.0180,  -0.9996,   0.0210,  -0.1056,  -0.0228,  -0.9941] 



# joint_positions:
#   - [0.02, -0.75, 0.016, -2.36, -0.01, 1.56, 0.77]  
#   - [0.02, -0.75, 0.016, -2.36, -0.01, 1.56, 0.87]
  # - [1.256, 1.225, -0.683, -1.33, -2.41, 1.19, 0.595]
  # publishing #4: std_msgs.msg.Float64MultiArray(layout=std_msgs.msg.MultiArrayLayout(dim=[], data_offset=0), data=[0.7, 0.0, 0.5, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0])



temperature_threshold: 10.0

home_joint_position: [0.0, -0.8, 1.2, 0.3, 0.0, 0.0]
